src/components/Onboarding/Screens/GardenTypeSelector.js

```diff
@@ -37,102 +41,141 @@ function GardenTypeSelector() {
+          Garden Type
         </h1>
         <div>
           <p
             css={`
-              ${textStyle('body1')};
+              ${textStyle('title4')};
               color: ${theme.contentSecondary};
             `}
           >
-            Choose which type of garden you'd like to launch.
+            Choose the type of garden you'd like to launch.
           </p>
         </div>
       </div>
       <div
         css={`
           display: flex;
           align-items: center;
-          justify-content: space-around;
-          margin-bottom: ${6 * GU}px;
+          justify-content: center;
         `}
       >
         <Card
           icon={defaultGardenLogo}
-          paragraph="Launch garden with new token"
+          paragraph="Create a new token that is native to your garden. "
           onSelect={handleSelectNative}
           selected={selectedType === NATIVE_TYPE}
-          title="Native"
+          animationData={nativeAnimation}
+          title="Native Token"
         />
         <Card
           icon={defaultGardenLogo}
-          paragraph="Launch garden with existing token"
+          paragraph="Use an existing ERC-20 token to within your garden."
           onSelect={handleSelectBYOT}
           selected={selectedType === BYOT_TYPE}
-          title="BYOT"
+          animationData={byotAnimation}
+          title="Pre-existing Token"
         />
       </div>
     </div>
   )
 }
 
-function Card({ icon, title, onSelect, paragraph, selected }) {
+function Card({
+  icon,
+  title,
+  onSelect,
+  paragraph,
+  selected,
+  animationData,
+  animationSegments,
+}) {
+  const [isPlaying, setIsPlaying] = useState(false)
+
   const theme = useTheme()
+
+  const handleStartAnimation = useCallback(() => {
+    setIsPlaying(true)
+  }, [setIsPlaying])
+
+  const handleStopAnimation = useCallback(() => {
+    setIsPlaying(false)
+  }, [setIsPlaying])
+
   return (
     <div
       css={`
         cursor: pointer;
-        padding: ${5 * GU}px;
+        padding-left: ${3.5 * GU}px;
+        padding-right: ${3.5 * GU}px;
+        padding-bottom: ${3.5 * GU}px;
+        margin: 10px;
+        max-width: 264px;
+        border-radius: 50% 50% ${2 * GU}px ${2 * GU}px;
         background: ${theme.surface};
         border: 1px solid ${theme.border};
         display: flex;
         flex-direction: column;
+        justify-content: center;
         align-items: center;
-        box-shadow: 0px 0px 4px 0px rgba(0, 0, 0, 0.1);
-        transition: box-shadow 0.3s ease;
+        transition: box-shadow 0.5s ease;
         ${selected
           ? `
-          box-shadow: 0px 0px 7px 1px ${theme.accentStart};
-          background: linear-gradient(
-            268deg,
-            ${theme.accentEnd},
-            ${theme.accentStart}
-          );
+          border: 2px solid #56D571;
+          transform: scale(1.02);
         `
           : `  &:hover {
           box-shadow: 0px 0px 7px 1px rgba(0, 0, 0, 0.2);
```

Comment:
Now that we highlight the card using the transform property and border highlight, we can probably reduce the radius of the shadow. What do you think?
```suggestion
          box-shadow: 0px 0px 2px 1px rgba(0, 0, 0, 0.2);
```